### PR TITLE
feat: add `palera1n` completion spec

### PR DESCRIPTION
**Pull request by itsnebulalol**
palera1n is a jailbreaking tool for iOS/iPadOS 15.x-16.x arm64.

https://github.com/palera1n/palera1n

### CODE REVIEW

**Comment by itsnebulalol** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/palera1n.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      description: "Boot fakefs",
+    },
+    {
+      name: ["--help", "-h"],
+      description: "Show help for palera1n",
+    },
+    {
+      name: ["--override-checkra1n", "-i"],
+      description: "Override checkra1n",
+      args: { name: "FILE" },
```
Comment (fedeci) [inline src/palera1n.ts:49]: Can you make this change for all the args that has a FILE type?
```suggestion
      args: { name: "FILE", suggestions: "filepaths" },
```

File: src/palera1n.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
+const completionSpec: Fig.Spec = {
+  name: "palera1n",
+  description: "Jailbreaking tool for iOS/iPadOS 15.x-16.x arm64",
+  options: [
+    {
+      name: ["--version", ""],
```
Comment (fedeci) [inline src/palera1n.ts:6]:
```suggestion
      name: "--version",
```

File: src/palera1n.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+      description: "Print version",
+    },
+    {
+      name: ["--force-revert", ""],
```
Comment (fedeci) [inline src/palera1n.ts:10]:
```suggestion
      name: "--force-revert",
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks @itsnebulalol! I left a couple suggestions that would be great to have committed before shipping this PR.

**Review by fedeci — APPROVED** [review]

**Comment by fedeci** [comment]
Thanks @itsnebulalol!

**Pull request merged**